src/v2/Apps/Settings/Routes/EditProfile/Components/__tests__/SettingsEditProfileAboutYou.jest.tsx

```diff
@@ -0,0 +1,61 @@
+import { screen, fireEvent, waitFor } from "@testing-library/react"
+import { graphql } from "react-relay"
+import { setupTestWrapperTL } from "v2/DevTools/setupTestWrapper"
+import { useUpdateMyUserProfile } from "v2/Utils/Hooks/Mutations/useUpdateMyUserProfile"
+import { SettingsEditProfileAboutYouFragmentContainer } from "../SettingsEditProfileAboutYou"
+
+jest.unmock("react-relay")
+jest.mock("v2/Utils/Hooks/Mutations/useUpdateMyUserProfile")
+
+const { renderWithRelay } = setupTestWrapperTL({
+  Component: SettingsEditProfileAboutYouFragmentContainer,
+  query: graphql`
+    query SettingsEditProfileAboutYou_Test_Query {
+      me {
+        ...SettingsEditProfileAboutYou_me
+      }
+    }
+  `,
+})
+
+describe("SettingsEditProfileAboutYou", () => {
+  const mockUseUpdateMyUserProfile = useUpdateMyUserProfile as jest.Mock
+  const mockSubmitUpdateMyUserProfile = jest.fn()
+
+  beforeEach(() => {
+    mockUseUpdateMyUserProfile.mockImplementation(() => ({
+      submitUpdateMyUserProfile: mockSubmitUpdateMyUserProfile,
+    }))
+  })
+
+  afterEach(() => {
+    mockUseUpdateMyUserProfile.mockReset()
+  })
+
+  it("renders the form", () => {
+    renderWithRelay()
+
+    expect(screen.getByText("Primary Location")).toBeInTheDocument()
+    expect(screen.getByText("Profession")).toBeInTheDocument()
+    expect(screen.getByText("Price Range")).toBeInTheDocument()
+  })
+
+  it("submits the form", async () => {
+    renderWithRelay()
+
+    fireEvent.change(screen.getByPlaceholderText("Profession"), {
+      target: { name: "profession", value: "Artist" },
+    })
+
+    await waitFor(() => {
```

Comment:
In the future you'll probably run into this again: with RTL, Formik's `onSubmit` needs to be waited on with `waitFor` otherwise you get an unmounted component update error.